aws/rust-runtime/aws-runtime/src/user_agent/interceptor.rs

```diff
@@ -99,6 +112,15 @@ impl Intercept for UserAgentInterceptor {
                 if let Some(app_name) = maybe_app_name {
                     ua.set_app_name(app_name.clone());
                 }
+
+                let maybe_connector_metadata = runtime_components
+                    .http_client()
+                    .and_then(|c| c.connector_metadata());
+                if let Some(connector_metadata) = maybe_connector_metadata {
+                    let am = AdditionalMetadata::new(Cow::Owned(connector_metadata.to_string()))?;
```

Comment:
I still don't think this follows the SEP the way we want. The SEP defines:

```
ua-pair              = ua-name ["#" ua-value]
ua-name              = token_no_hash
ua-value             = token
...
additional-metadata  = "md/" ua-pair
```

I would think we want this to show up as `md/http#hyper-1.x` or `md/http#hyper-0.x`, right now I think it shows up as just `md/hyper-1.x`.